meta-genivi-dev/recipes-sota/rvi-sota-client/rvi-sota-client_git.bb

```diff
@@ -112,10 +112,9 @@ SRC_URI[index.sha256sum] = "86114b93f1f51aaf0aec3af0751d214b351f4ff9839ba031315c
 
 SYSTEMD_SERVICE_${PN} = "sota_client.service"
 
-DEPENDS += " openssl "
+DEPENDS += " openssl dbus "
```

Comment:
Sorry, I do not understand what is not clear. I will try to answer to your questions and provide more details.
The change is needed because otherwise in certain cases the build of RVI SOTA Client may fail which will lead to failure of building the whole GDP image. I noticed this issue recently. For long time dbus existed as a run-time dependency (RDEPENDS) and now I have just moved it to DEPENDS. In my opinion it is a race condition because if something else bitbakes dbus before building RVI SOTA client the issue will not occur.